lib/http/2/connection.rb

```diff
@@ -350,8 +350,9 @@ def receive(data)
         end
       end
 
+    rescue Error::Error
+      raise
     rescue => e
-      raise if e.is_a?(Error::Error)
```

Comment:
?

Reply:
Exceptions already match based on class, so it seems like a code smell to do this explicitly.